Assets/02_Student Folders/SaidKrebbers_Assets/Prefabs/Enemies/Enemy_Turret.prefab

```diff
@@ -0,0 +1,1554 @@
+%YAML 1.1
```

Comment:
Why did you make a copy of the enemy prefabs to your folder? 
I dont see a reason in your commit comments or summaries

Reply:
I made copies of the enemies because I changed multiple things about them and did not want these changes to affect the others.

Reply:
actually i just noticed i added half of the changes to the specific enemies in my lvls and not the prefabs themselves.
And some to the prefabs. which is slightly messy..., but oke for this assignment i believe